taskslist/app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Edit a Task</h1>
+
+<%= render partial: 'form', locals: {method_type: :patch, button_text: "Update"} %>
```

Comment:
Is the `method_type` variable used anywhere in the form partial? I saw another student using this same variable in their locals, but they also were not explicitly using it. Is this some kind of special variable that the `form_for` helper knows to use?